docs/pages/example/snapshot.html

```diff
@@ -0,0 +1,102 @@
+<style>
+  html, body {
+    margin: 0;
+  }
+
+  #map {
+    position: absolute;
+    width:50%;
+    height:100%;
+    background: rgba(0,255,0,0.5)
+  }
+
+  #snapshot {
+    position: absolute;
+    left:50%;
+    background:rgba(0,0,255,0.5)
+  }
+
+  h1 {
+    color:white;
+    text-align:center;
+    position: relative;
+    top:50%;
+    transform: translateY(-50%);
+    margin:0;
+  }
+ 
+</style>
+
+<div id='map' class='window'></div>
+
+<canvas id='snapshot' class='window'></div>
+
+<script>
+
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/dark-v9',
+    center: {
+        lat: 37.4036,
+        lng: -122.0325
+    },
+    zoom: 10,
+    preserveDrawingBuffer: true// not guaranteed to work if false
```

Comment:
I would just add a comment like " required for the map's canvas to be exported to a PNG"

Reply:
I have updated the comment to this suggestion.